functions/toml.php

```diff
@@ -0,0 +1,93 @@
+<?php
+/*
+TOML Parser
+Super inefficient, but gets the job done.
+Limitation: only gets the first dependency. TODO: split dependency.* into it's own array, and stuff the * into their own id within dependency to prevent overwrite.
```

Comment:
Dependencies currently do not really matter, unless this PR implements a UI element for dependencies. Currently, I believe it is unused, but is for future-proofing.